src/plugins/chart_expressions/expression_metric/public/components/metric_component.tsx

```diff
@@ -112,11 +110,11 @@ class MetricVisComponent extends Component<MetricVisComponentProps> {
     const config = this.props.visParams.metric;
     const dimensions = this.props.visParams.dimensions;
     const isPercentageMode = config.percentageMode;
-    const min = config.colorsRange[0].from;
-    const max = (last(config.colorsRange) as Range).to;
+    const min = config.colorsRange?.[0]?.from ?? 0;
+    const max = last(config.colorsRange)?.to ?? 0;
```

Comment:
It can be the case, where `colorRange` array is empty. Which value should you assign to the max then? I don't have a better idea, as if later it is sorting by this value. I'd add `-Infinity` if I could)))

Reply:
I think that we can never have zero ranges. At least from the UI I see that there is always one, and I can't remove it. Right? I would default it to 1 to be honest. Also in some lines below we are dividing using max-min. Let's avoid this to be set to 0.

Reply:
Done.